src/js/stores/MesosStateStore.js

```diff
@@ -128,20 +128,15 @@ var MesosStateStore = Store.createStore({
     let services = this.get('lastMesosState').frameworks;
     let foundTask = null;
 
-    for (let serviceIndex in services) {
-      let service = services[serviceIndex];
+    services.some(function (service) {
```

Comment:
is it just me or before we were doing `for.. in` which is for objects, and here we are doing `.some` which is for arrays.

am i missing something?